programs/vote/src/vote_state/mod.rs

```diff
@@ -241,6 +241,10 @@ impl VoteStateUpdate {
     pub fn slots(&self) -> Vec<Slot> {
         self.lockouts.iter().map(|lockout| lockout.slot).collect()
     }
+
+    pub fn last_voted_slot(&self) -> Option<Slot> {
+        self.lockouts.back().map(|l| l.slot)
+    }
```

Comment:
For sake of paper trail, this was the extra bit needed for v1.14 BP in comparison to original PR